### DISCUSSION TITLE
Soru işaretli yere gelecek sayı nedir?

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#73] [Zeka Sorusu] [answered]
8621485265 = 7
1458715613 = 4
8913232845 = 6
9846332124 = ?

Bağıntıya göre soru işaretli yere gelecek sayı nedir

**Comment by Berkek007** [comment, marked as answer]
Cevap :6, çünkü rakamların şekillerinden gidecek olursak 8 rakımı 2 boşluklu, 6 ve 9 rakamı 1 boşluklu, 4 rakamı da bir boşluklu, 1,2,3,5,7 rakamlarında ise boşluk 0. O yüzden ilkinde 8,6,4,8 ve 6 rakamlarının boşluklarını toplarsak 7 rakamına ulaşırız. Aynı şekilde ikincisinde 4,8,6 rakamlarının boşuklarını toplarsak 4 rakamına ulaşırız. Bu sebeple de bize sorulan soru işaretli yere 9,8,4,6,4 rakamlarının boşluklarını toplayarak 6 rakamına ulaşırız.
